core/src/main/scala/org/bitcoins/core/util/Bech32.scala

```diff
@@ -305,6 +305,17 @@ object Bech32 extends Bech32 {
   val charset: Vector[Char] = Vector('q', 'p', 'z', 'r', 'y', '9', 'x', '8',
     'g', 'f', '2', 't', 'v', 'd', 'w', '0', 's', '3', 'j', 'n', '5', '4', 'k',
     'h', 'c', 'e', '6', 'm', 'u', 'a', '7', 'l')
+
+  /** The Bech32 character set for decoding. */
```

Comment:
is there a reference to this? Is it reversed or the inverse?

Reply:
Added link from where I got it from